Task_DEV_5/Task_DEV_5/FirstSorter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.IO;
+
+namespace task_DEV_5
+{
+    class FirstTextSorter
```

Comment:
Bad class name. It is not understandable from the context what means FirstSorter and SecondSorter.

Reply:
Done.